### ISSUE TITLE
Control candlestick width

### ISSUE THREAD

**Issue by camstuart** [tradingview/lightweight-charts#792]
Hi,

Fantastic charting library by the way!

Is it possible to control the width of a candlestick? My use case is that I am using the candlestick chart type to display a renko brick chart. 

Ideally, the height of my bricks (which is calculated prior to rendering) would be the same as the width, giving a more "typical renko brick" appearance.

Thanks

**Comment by timocov** [comment]
Merged to #272

**Comment by nick-chandoke** [comment]
If merging this with issue #272 is to suggest that renko should be implemented as a plugin, I ask for reconsideration since renko always have a certain width. Namely if I redefine
```
optimalCandlestickWidth(barSpacing,pixelRatio) {
  return Math.floor(pixelRatio*(barSpacing+1));
}
``` then my candle series plots like renko. Seemingly, less code is needed to add a "renko" option to `CandleStickStyleOptions` than to create a new plugin, and of course also it would benefit everyone who wants to use renko out-of-the-box.
